Godeps/Godeps.json

```diff
@@ -11,19 +11,10 @@
 			"Rev": "75cd24fc2f2c2a2088577d12123ddee5f54e0675"
 		},
 		{
-			"ImportPath": "code.google.com/p/gcfg",
```

Comment:
This seems strange.  I see the `json` file updated, but I don't see any updated files.  Mistaken commit somewhere?